website/cue/reference/components/sources/prometheus_pushgateway.cue

```diff
@@ -0,0 +1,104 @@
+package metadata
+
+components: sources: prometheus_pushgateway: {
+	title: "Prometheus Pushgateway"
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+		stateful:      false
+	}
+
+	features: {
+		auto_generated:   true
+		acknowledgements: true
+		multiline: enabled: false
+		receive: {
+			from: {
+				service: services.prometheus
+
+				interface: socket: {
+					api: {
+						title: "Prometheus Pushgateway"
+						url:   urls.prometheus_pushgateway
```

Comment:
This `urls.prometheus_pushgateway` is referring to an entry in https://github.com/vectordotdev/vector/blob/master/website/cue/reference/urls.cue. But currently there is no such entry for `prometheus_pushgateway`. Would you be able to add an entry to point to https://github.com/prometheus/pushgateway/blob/master/README.md.

I'll be honest and admit that I'm not sure if this information is actually used anywhere in our documentation, but I could be missing something!